tests/data/_IsmrmrdRawTestData.py

```diff
@@ -221,12 +232,12 @@ def __init__(
 
         # Loop over the repetitions, add noise and write to disk
         for rep in range(self.repetitions):
-            noise = self.noise_level * torch.randn(self.ncoils, nky // self.acceleration, nkx, dtype=torch.complex64)
+            noise = self.noise_level * torch.randn(self.ncoils, nkx, nky // self.acceleration, dtype=torch.complex64)
             # Here's where we would make the noise correlated
             K = ktrue + noise
             acq.idx.repetition = rep
-            for idx, line in enumerate(ky_idx):
-                if not self.flag_invalid_reps or rep == 0 or idx < len(ky_idx) // 2:  # fewer lines for rep > 0
+            for idpe, line in enumerate(pe_line_idx):
```

Comment:
Here the naming of pe_line_idx, idpe, line, line_idx is a maybe bit confusing (or I am tired)..

Reply:
Rewrote that part a bit, hope it is clearer now.